ObsCoreExtensionForRadioData.tex

```diff
@@ -268,16 +267,18 @@ \subsection{s\_fov}
 %mid value of the spectral range
 receiver nominal wavelength and D coincides with the telescope diameter (SD case) or the largest diameter of the array antennae or telescopes (interferometric case).
 In interferometry, the correlator can also restrict the fov depending on the trade-off set to build the signal. 
-
+Nominal wavelength SHOULD be taken as the mid value of the spectral range except if data providers have good reasons to propose another value which should be documented in the FIELD DESCRIPTION tag in that case. 
```

Comment:
How the nominal wavelength is defined in the data collection should be clarified if we want to query and compare on s_resolution or s_fov . 
we would  have to add  a new item in the TAP radio extension to mention it , like em_nominal_wavelength for instance , 
em_nominal_wavelength , in meters, with ucd =em.wl.effective . 
Is it necessary , or can we rely instead on the min and max values for all features depending on the spectral range : s_fov, s_largest_max scale and s_resolution ???

Reply:
> How the nominal wavelength is defined in the data collection should be clarified if we want to query and compare on s_resolution or s_fov . we would have to add a new item in the TAP radio extension to mention it , named em_nominal_wavelength for instance , with em_nominal_wavelength , in meters, with ucd =em.wl.effective . Is it necessary ? Or can we rely instead , in the radio extension table, on the min and max values for all features depending on the spectral range : s_fov, s_largest_max scale and s_resolution ???
?For sure, nominal wavelength is used to estimate nominal s_fov and nominal s_resolution. But although I think  queries on these nominal s_fov and s_resolution are useful for coarse grain discovery , I don't think we have demands for discovery using this nominal wavelength.   So is it useful to add it in the list of attributes ? Thoughts ?